packages/manager/apps/web/client/app/components/service-status/service-status-action.html

```diff
@@ -38,13 +38,21 @@
     </dl>
     <oui-action-menu data-compact data-placement="end">
         <oui-action-menu-item
-            data-ng-if="$ctrl.isExpired() || !$ctrl.isInAutoRenew()"
+            data-ng-if="$ctrl.serviceInfos.state ==='expired' && ($ctrl.isExpired() || !$ctrl.isInAutoRenew())"
```

Comment:
I guess we could pproced with some refactoring to make code more cleanup:
```suggestion
            data-ng-if="$ctrl.isServiceInfosExpired() && ($ctrl.isExpired() || !$ctrl.isInAutoRenew())"
            
            Into controller:
            isServiceInfosExpired() {
               this.serviceInfos.state === SERVICE_INFO_STATS.EXPIRED;
            }
            
            Into constant file:
            export default SERVICE_INFO_STATS = {
              EXPIRED : 'expired',
              RESTORABLE: 'restorable',
              ..........(to complete)
            }
```